host/4/bullseye/amd64/python/python310/python310.Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apt-get update && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.10-slim-bullseye
+FROM mcr.microsoft.com/mirror/docker/library/python:3.10-slim
```

Comment:
This image stayed slim.  Which is good. Please keep this consistency in all non-suffix images.